Source/VCGeneration/VCGen.cs

```diff
@@ -489,13 +490,13 @@ void DFS(Block cur)
 
           if (needToCheck)
           {
-            CheckUnreachable(cur, seq);
+            CheckUnreachable(cur, seq, cancellationToken);
```

Comment:
Same here, isn't there `this.cancellationToken`?

Reply:
It was no longer needed anyway, so I removed it.